docker/docker-compose.yml

```diff
@@ -1,10 +1,9 @@
-version: "3.8"
 services:
     app:
       extends: 
-        file: docker-compose-generic.yml
+        file: docker-compose-base.yml
         service: app
-      image: ghcr.io/openboxes/openboxes:feature-upgrade-to-grails-3.3.10
+      image: ghcr.io/openboxes/openboxes:v0.9.3
```

Comment:
would we want this to be "latest"? Otherwise we'd need to make sure we update this each release

Reply:
@EWaterman After the initial approval on my end, I started wondering which one should actually be here. `Develop` or `Main`. I was leaning to develop, but thought we could change it later on our own, once it is merged.